plugins/Feedback/Feedback.php

```diff
@@ -171,22 +113,35 @@ public function getShouldPromptForFeedback()
 
         $feedbackReminder = new FeedbackReminder();
         $nextReminderDate = $feedbackReminder->getUserOption();
+        $now = Date::now()->getTimestamp();
 
+        //user answered question
         if ($nextReminderDate === self::NEVER_REMIND_ME_AGAIN) {
             return false;
         }
 
+        // if is new user or old user field not exist
         if ($nextReminderDate === false) {
```

Comment:
If we remove the never remind me option check above, but still use the old option, we actually need to handle that here I guess
```suggestion
        if ($nextReminderDate === false || $nextReminderDate <= 0) {
```